app/__main__.py

```diff
@@ -11,24 +12,28 @@
 parser = argparse.ArgumentParser(
     prog=__package__,
     description="preview raw data as an image of chosen format")
-parser.add_argument("FILE_PATH", help="file containing raw image data")
+parser.add_argument("-f",
+                    "--FILE_PATH",
+                    help="file containing raw image data",
+                    default=None)
 parser.add_argument("-c",
                     "--color_format",
                     choices=AVAILABLE_FORMATS.keys(),
                     default=list(AVAILABLE_FORMATS.keys())[0],
                     help="target color format (default: %(default)s)")
 parser.add_argument("-r",
                     "--resolution",
-                    metavar=("width", "height"),
+                    metavar=("width"),
                     type=int,
-                    nargs=2,
-                    default=[600, 600],
+                    nargs=1,
+                    default=600,
                     help="target resolution (default: %(default)s)")
```

Comment:
Here's an instance of when used should be asked to give width instead of full resolution.